servers/dsub/jobs/test/base_test_cases.py

```diff
@@ -25,18 +25,21 @@
 from jobs.models.aggregation_response import AggregationResponse
 from jobs.models.status_counts import StatusCounts
 
-DOCKER_IMAGE = 'ubuntu:14.04'
+DOCKER_IMAGE = 'ubuntu:20.04'
```

Comment:
Curious to know if these tests work. However, according to the README:
> dsub support is deprecated and will be removed in a future release.

Reply:
it looks like the dsub tests were actually [disabled last year](https://github.com/DataBiosphere/job-manager/pull/726/files)